sunshine/config.cpp

```diff
@@ -573,6 +573,10 @@ int apply_flags(const char *line) {
 }
 
 void apply_config(std::unordered_map<std::string, std::string> &&vars) {
+  if(!fs::exists(stream.file_apps.c_str())) {
+      stream.file_apps = SUNSHINE_DEFAULT_DIR "/" APPS_JSON;
```

Comment:
I like the idea of having a default apps file in the event that the config file the user wants not yet exists.

However, later on you'll overwrite what the user wanted. I think a better method would be:

If `apps_file` doesn't exist, copy contents of default apps file to `apps_file`: This will allow the end user to configure the path to the config file.